packages/protocol/contracts/identity/interfaces/IAttestations.sol

```diff
@@ -16,12 +16,13 @@ interface IAttestations {
 
   // TODO: For some reason the compiler will complain about the next line
   // function setAccountDataEncryptionKey(bytes) external;
+  // function setMetadataURL(string calldata) external;
```

Comment:
Why is this commented out?

Reply:
The compiler complained about non matching byte codee, same with setting the encryption key ...